### PR TITLE
Eslint to project

### PR DESCRIPTION
**Pull request by Omotola28**
Installed eslin to project 
Fixed eslint suggestions in App.js and serviceworker.js

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  ],
+  "plugins": [
+    "jsx-a11y",
+    "prettier"
+  ],
+  "rules": {
+    "semi": 0,
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+  }
+}
```
Comment (mazma1) [inline .eslintrc]: This is missing a new line at the end so the file can breathe..

File: package.json [hunk 2]
```diff
@@ -8,13 +8,15 @@
     "@testing-library/user-event": "^7.1.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "typescript": "^3.9.6"
```
Comment (mazma1) [inline package.json]: Do we need TypeScript for anything?

**Review by mazma1 — COMMENTED** [review]
@Omotola28 this is a good configuration to start with. I left a few comments; nothing much and we should be good to go

File: .eslintrc (hunk 1 above)
Comment (Omotola28) [reply .eslintrc]: Alright I would add one and send this back up

File: package.json (hunk 2 above)
Comment (Omotola28) [reply package.json]: No we do not must have mistakenly added it to the package.json file

**Review by mazma1 — APPROVED** [review]

**Pull request merged**